packages/core/src/components/read-more/read-more.scss

```diff
@@ -0,0 +1,54 @@
+@use "../../scss/vars/colours";
+@use "../../scss/vars/typography";
+
+section.expansion {
+  padding: 2px 0;
+
+  .expansion-content {
+    display: none;
+    max-height: 0;
+    overflow: visible;
+    border-left: 4px solid colours.$colour-utility-info;
+  }
+  &.expanded .expansion-content {
+    display: block;
+    max-height: initial;
+    padding: 2px 5px;
+  }
+
+  button {
+    background-color: transparent;
+    border: none;
+    white-space: unset;
+    color: initial;
+    cursor: pointer;
+    margin: 8px 0px;
+    padding: 4px;
+    align-items: baseline;
+    text-align: left;
+    text-decoration: underline;
+
+    &.expanded {
+      font-weight: typography.$font-weight-medium;
+    }
+
+    &::before{
+      content: url("svg/mat_fwd_arrow_icon.svg");
```

Comment:
Can we use the ukho-icon component please? Don't worry about it using the font-awesome version, we will be switching to material icons in the future